io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/InstanceExportConfigurationPage.java

```diff
@@ -110,6 +111,13 @@ protected void createContent(Composite page) {
 		final Label label = new Label(page, SWT.NONE);
 		label.setText("Choose your Type you want to export:");
 
+		Label separatorLabel = new Label(page, SWT.NONE);
+		separatorLabel.setText("Warning! Feature types with no data are not selectable");
+
+		// Set the text colour of the label to yellow
+		Color yellow = PlatformUI.getWorkbench().getDisplay().getSystemColor(SWT.COLOR_DARK_GRAY);
```

Comment:
Dark grey results in yellow?

Reply:
:-) I'll rename it. It was yellow at the beginning but was not seen nicely so I have change it to grey.